#### 참고: https://wikidocs.net/157997
#### 정답: https://wikidocs.net/157712#07
#### 날짜: 21-01-24

## Q1
- - -
```java
import HousePark;
------------------------
import house.HousePark;
```

## Q2
- - -
result = 7
> 배열의 3번째 원소는 없다.  
> ArrayIndexOutOfBoundsException 캐치 + finally 실행

## Q3
- - -
Thread를 상속받아서 진행.  
근데 join 할 때 왜 try catch 를 써야하는거지??? (안쓰니까 checked Exception 발생.)

### 참고: http://happinessoncode.com/2017/10/09/java-thread-interrupt/

## Q4
- - -
강의에서는 왜 boxed를 쓴거지????

## Q5
- - -
easy